demos/continuous_batching/agentic_ai/README.md

```diff
@@ -0,0 +1,143 @@
+```
+:::
+
+:::{tab-item} GPU
+
+In case you want to use GPU device to run the generation, add extra docker parameters `--device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1)`
+to `docker run` command, use the image with GPU support. Export the models with precision matching the GPU capacity and adjust pipeline configuration.
+It can be applied using the commands below:
+```bash
+docker run -d --rm -p 8000:8000 --device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -v $(pwd)/models:/models:ro openvino/model_server:2025.2-gpu \
+--rest_port 8000 --model_path /models/Qwen/Qwen3-8B --model_name Qwen/Qwen3-8B
+```
+:::
+:::{tab-item} NPU
+
+Running this command starts the container with NPU enabled:
+```bash
+docker run -d --rm --device /dev/accel --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) \
+-p 8000:8000 -v $(pwd)/models:/models:rw openvino/model_server:latest-gpu --rest_port 8000 --model_path /models/Qwen/Qwen3-8B --model_name Qwen/Qwen3-8B
+```
+:::
+
+::::
+
+
+**Deploying on Bare Metal**
+
+Assuming you have unpacked model server package, make sure to:
+
+- **On Windows**: run `setupvars` script
+- **On Linux**: set `LD_LIBRARY_PATH` and `PATH` environment variables
+
+as mentioned in [deployment guide](../../docs/deploying_server_baremetal.md), in every new shell that will start OpenVINO Model Server.
+
+Depending on how you prepared models in the first step of this demo, they are deployed to either CPU, GPU or NPU (it's defined in `graph.pbtxt`). If you run on GPU or NPU, make sure to have appropriate drivers installed, so the device is accessible for the model server.
+
+```bat
+ovms --rest_port 8000 --model_path /models/Qwen/Qwen3-8B --model_name Qwen/Qwen3-8B
+```
+
+
+## Start MCP server with SSE interface
+
+### Linux
+```bash
+git clone https://github.com/isdaniel/mcp_weather_server
+cd mcp_weather_server
+docker build . -t mcp_weather_server
+docker run -d -v $(pwd)/src/mcp_weather_server:/mcp_weather_server  -p 8080:8080 mcp_weather_server bash -c ". .venv/bin/activate ; python /mcp_weather_server/server-see.py"
+```
+
+> **Note:** On Windows the MCP server will be demonstrated as an instance with stdip interface inside the agent application
+
+## Start the agent
+
+Install the application requirements
+
+```console
+curl https://raw.githubusercontent.com/openvinotoolkit/model_server/agentic-demo/demos/continuous_batching/agentic_ai/openai_agent.py -o openai_agent.py
+pip install -r https://raw.githubusercontent.com/openvinotoolkit/model_server/agentic-demo/demos/continuous_batching/agentic_ai/requirements.txt
+```
+
+Run the agentic application
+```console
+python openai_agent.py --query "What is the weather now in Tokyo?"
+
+python openai_agent.py --query "What is the content of my local folder?"
```

Comment:
In the previous step we deploy weather service. Will it give reasonable response to such question?